src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const styles = {};
+
+  sourceString.split(';').map(line =>
+    line.trim()).filter(line => line)
+    .map(lns => {
+      const [property, value] = lns.split(':');
```

Comment:
I think **lns** is a bit unclear variable name. I liked the usage of destructuring.